usr/share/rear/backup/NETFS/default/41_save_capabilities.sh

```diff
@@ -0,0 +1,11 @@
+# save all found capapilities to file 
+
+if [ "$BACKUP_CAP" == "y" ] ; then
+	Log "Saving capabilities to File." 
+	if which getcap >/dev/null 2>&1 ; then
+		getcap -r / 2>/dev/null | grep -v $ISO_DIR > $VAR_DIR/recovery/capabilities
+	else
+		Log "getcap binary not found."
```

Comment:
I am against having code in different places that checks things that we already check somewhere else. If you put something into REQUIRED_PROGS then we have one place to validate that. All other places should assume that the prog is there without extra checks.